src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/SemanticTokensEncoderTests.cs

```diff
@@ -5,6 +5,8 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Numerics;
+using System.Reflection.Emit;
```

Comment:
Seems like unused imports. make sure to go thru unused imports. Happens a lot with visual studio